electrum/three_keys/utils.py

```diff
@@ -0,0 +1,41 @@
+from typing import Tuple
+
+from .tx_type import TxType
+
+
+TX_STATUS_INDEX_SHIFT = 10
+TX_TYPES_LIKE_STANDARD = (
+    TxType.NONVAULT,
+    TxType.INSTANT,
+    TxType.ALERT_CONFIRMED,
+)
+
+
+def update_tx_status(tx_hash, tx_mined_info: 'TxMinedInfo', status: int, status_str: str, db) -> Tuple[int, str]:
+    if status_str == 'unknown':
+        return status, status_str
+
+    tx = db.get_transaction(tx_hash)
+
+    if tx.tx_type in TX_TYPES_LIKE_STANDARD:
+        if tx.tx_type == TxType.ALERT_CONFIRMED:
```

Comment:
shouldn't `ALERT_RECOVERED` be handled?

Reply:
`ALERT_RECOVERED` is handled in return statement of this method, like other transaction types